Coding-Tracker.Lawang/Lawang.Coding-Tracker.generated.sln

```diff
@@ -0,0 +1,25 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 17
+VisualStudioVersion = 17.5.002.0
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Lawang.Coding-Tracker", "Coding-Tracker.Lawang\Lawang.Coding-Tracker.csproj", "{617C5177-5C40-4539-A3E9-C1CF01286336}"
```

Comment:
I think you must have changed the directory structure as this solution cannot be loaded as is.
I had to change to this in order to load the solution:
```Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Lawang.Coding-Tracker", "Coding-Tracker.Lawang\Lawang.Coding-Tracker.csproj", "{617C5177-5C40-4539-A3E9-C1CF01286336}"```
to
```Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Lawang.Coding-Tracker", "Lawang.Coding-Tracker.csproj", "{617C5177-5C40-4539-A3E9-C1CF01286336}"```